config/manager/manager.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1alpha1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
```

Comment:
let's change control-plane label to something like `eraser.sh/system: yes` (change in all locations)